reana_cluster/backends/kubernetes/templates/deployments/job-controller-template.yaml

```diff
@@ -48,14 +57,32 @@ spec:
         {% endfor %}
       volumes:
       - name: reana-shared-volume
-        {% if DEPLOYMENT == 'prod' %}
+        {% if CEPHFS %}
         persistentVolumeClaim:
           claimName: manila-cephfs-pvc
           readOnly: false
         {% else %}
         hostPath:
           path: {{ROOT_PATH}}
         {% endif %}
+        {% if CVMFS %}
```

Comment:
Do we need to mount CVMFS on RJC?

Reply:
It's not necessary but it might come useful, checking directory contents for modifications comes to mind